examples/actix-example.rs

```diff
@@ -0,0 +1,148 @@
+use actix_web::{web, App, HttpResponse, HttpServer, Responder};
```

Comment:
This adds the initial structure of the testing endpoint. Some of this changes in a later commit was the SSE test harness was changed quite a bit but the general concept is still reasonable.